docs/usage.md

```diff
@@ -510,3 +510,522 @@ This can be replicated for ROCm for AMD , by following the [instructions to set
 
 See the [MLCube documentation](https://mlcommons.github.io/mlcube/) for more details.
+
+## Hugging Face Hub CLI
+
+This tool allows you to interact with the Hugging Face Hub directly from a terminal. For example, you can login to your account, create a repository, upload and download files, etc. It also comes with handy features to configure your machine or manage your cache.
+
+### huggingface-cli login
+
+In many cases, you must be logged in to a Hugging Face account to interact with the Hub (download private repos, upload files, create PRs, etc.). To do so, you need a [User Access Token](https://huggingface.co/docs/hub/security-tokens) from your [Settings page](https://huggingface.co/settings/tokens). The User Access Token is used to authenticate your identity to the Hub. Make sure to set a token with write access if you want to upload or modify content.
+
+Once you have your token, run the following command in your terminal:
+
+```bash
+>>> gandlf hf login
+```
+
+This command will prompt you for a token. Copy-paste yours and press *Enter*. Then you'll be asked if the token should also be saved as a git credential. Press *Enter* again (default to yes) if you plan to use `git` locally. Finally, it will call the Hub to check that your token is valid and save it locally.
+
+```
+_|    _|  _|    _|    _|_|_|    _|_|_|  _|_|_|  _|      _|    _|_|_|      _|_|_|_|    _|_|      _|_|_|  _|_|_|_|
+_|    _|  _|    _|  _|        _|          _|    _|_|    _|  _|            _|        _|    _|  _|        _|
+_|_|_|_|  _|    _|  _|  _|_|  _|  _|_|    _|    _|  _|  _|  _|  _|_|      _|_|_|    _|_|_|_|  _|        _|_|_|
+_|    _|  _|    _|  _|    _|  _|    _|    _|    _|    _|_|  _|    _|      _|        _|    _|  _|        _|
+_|    _|    _|_|      _|_|_|    _|_|_|  _|_|_|  _|      _|    _|_|_|      _|        _|    _|    _|_|_|  _|_|_|_|
+
+To login, `huggingface_hub` requires a token generated from https://huggingface.co/settings/tokens .
+Token:
+Add token as git credential? (Y/n)
+Token is valid (permission: write).
+Your token has been saved in your configured git credential helpers (store).
+Your token has been saved to /home/wauplin/.cache/huggingface/token
+Login successful
+```
+
+Alternatively, if you want to log-in without being prompted, you can pass the token directly from the command line. To be more secure, we recommend passing your token as an environment variable to avoid pasting it in your command history.
+
+```bash
+# Or using an environment variable
+>>> gandlf hf login --token $HUGGINGFACE_TOKEN --add-to-git-credential
+Token is valid (permission: write).
+Your token has been saved in your configured git credential helpers (store).
+Your token has been saved to /home/wauplin/.cache/huggingface/token
+Login successful
+```
+
+For more details about authentication, check out [this section](../quick-start#authentication).
```

Comment:
There seem to be links starting with `../`, which I think are not going to be present since they are referring to HF documentation. Please link to the proper HTML page from HF for such links.

For example, `../quick-start#authentication` should be https://huggingface.co/docs/huggingface_hub/v0.23.4/en/quick-start#authentication